File: 0832.翻转图像/solution.go
func flipAndInvertImage(A [][]int) [][]int {
	for i := range A {
		l, r := 0, len(A[i])-1
		for l <= r {
			A[i][l], A[i][r] = A[i][r]^1, A[i][l]^1
			l++
			r--
		}
	}
	return A
}

File: 0832.翻转图像/solution.cpp
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& A) {
        int row=A.size();
        int col=A.at(0).size();
        for (int i=0;i<row;i++) {
            int l=0,r=col-1;
            while (l<=r) {
                int a=A.at(i).at(l),b=A.at(i).at(r);
                A.at(i).at(l)=b^1;
                A.at(i).at(r)=a^1;
                l++;
                r--;
            }
        }
        return A;
    }
};
